src/tags/for_block.rs

```diff
@@ -98,14 +99,41 @@ fn iter_slice(
     slice
 }
 
+/// Extracts an integer value or an identifier from the token stream
+fn parse_attr(args: &mut Iter<Token>) -> Result<Argument> {
+    expect(args, &Token::Colon)?;
+    match args.next() {
+        t @ Some(Token::IntegerLiteral(_)) | t @ Some(Token::Identifier(_)) => {
+            value_token(t.unwrap().clone())?.to_arg()
```

Comment:
Added a note to https://github.com/cobalt-org/liquid-rust/issues/95 to make `value_token` work by reference to avoid all these unnecessary clones (this and elsewhere)